arkeocli/claim.go

```diff
@@ -69,15 +69,15 @@ func runClaimCmd(cmd *cobra.Command, args []string) (err error) {
 	}
 
 	clientPubkey := contract.GetDelegate()
-	if clientPubkey == "" {
+	if clientPubkey.IsEmpty() {
 		clientPubkey = contract.GetClient()
 	}
 
 	creatorAddr, err := clientPubkey.GetMyAddress()
 	if err != nil {
 		return err
 	}
-	creator := creatorAddr.String()
+	creator := creatorAddr
```

Comment:
not sure I am seeing why this assignment is needed now?

Reply:
yup, removed.